crates/eldenring/src/cs/game_man.rs

```diff
@@ -0,0 +1,201 @@
+    unkafc: [u8; 0x4],
+    unkb00: i32,
+    unkb04: F32Vector3,
+    unkb10: F32Vector4,
+    unkb20: BlockId,
+    unkb24: u32,
+    unkb28: bool,
+    unkb2c: i32,
+    unkb30: F32Vector3,
+    unkb40: F32Vector4,
+    unkb50: BlockId,
+    unkb54: u32,
+    unkb58: bool,
+    unkb59: u8,
+    /// Whether or not item replenishment from chests is requested.
+    /// Usually set when sitting at a Site of Grace.
+    pub item_replanish_from_chest_requested: bool,
+    /// Whether or not item restoration after quickmatch is requested. \
+    /// See [crate::cs::EquipGameData::qm_item_backup_vector]
+    pub item_restore_after_qmrequested: bool,
+    unkb5c: u8,
+    unkb5d: u8,
+    unkb5e: u8,
+    unkb5f: u8,
+    unkb60: u32,
+    unkb64: u32,
+    unkb68: u32,
+    unkb6c: u32,
+    pub new_game_plus_requested: bool,
+    unkb71: u8,
+    pub save_requested: bool,
+    unkb73: u8,
+    unkb74: u8,
+    unkb75: u8,
+    /// Save slot index requested for loading.
+    pub requested_save_slot_load_index: i32,
+    unkb7c: u8,
+    unkb7d: u8,
+    pub save_state: u32,
+    unkb88: DLDateTime,
+    unkb98: DLDateTime,
+    unkba8: DLDateTime,
+    unkbb8: u32,
+    unkbbc: u32,
+    unkbc0: u32,
+    unkbc4: u32,
+    pub is_in_online_mode: bool,
+    unkbc9: u8,
+    pub event_world_type: EventWorldType,
+    unkbcb: u8,
+    unkbcc: u8,
+    unkbcd: u8,
+    unkbce: u8,
+    unkbcf: [u8; 0x19],
+    unkbe8: u64,
+    unkbf0: [u8; 0x10],
+    /// See [crate::cs::CSStayInMultiplayAreaWarpData]
+    pub stay_in_multiplay_area_saved_position: F32Vector3,
+    /// See [crate::cs::CSStayInMultiplayAreaWarpData]
+    pub stay_in_multiplay_area_saved_block_id: BlockId,
+    /// See [crate::cs::CSStayInMultiplayAreaWarpData]
+    pub stay_in_multiplay_area_saved_rotation: F32Vector4,
+    unkc20: [u8; 0x20],
+    unkc40: F32Vector4,
+    unkc50: F32Vector4,
+    pub sub_area_name_popup_message_id: i32,
+    pub update_task: CSEzUpdateTask<CSEzTask, Self>,
+    unkc90: [u8; 0xf4],
+    pub summon_param_type: SummonParamType,
+    pub multiplay_role: MultiplayRole,
+    pub has_password: bool,
+    pub party_member_info: OwnedPtr<PartyMemberInfo>,
+    unld98: [u8; 0xd8],
+    pub character_name_is_empty: bool,
+}
+
+#[repr(u8)]
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
+pub enum EventWorldType {
+    Local = 0,
+    Remote = 1,
+}
+
+#[repr(u32)]
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
+pub enum ForceCamRotationMethod {
+    Default = 0,
+    Closest = 1,
+    CounterClockwise = 2,
+    Clockwise = 3,
+}
+
+impl FromStatic for GameMan {
+    unsafe fn instance() -> fromsoftware_shared::InstanceResult<&'static mut Self> {
+        use crate::rva;
+        use pelite::pe64::Pe;
+
+        let target = Program::current()
+            .rva_to_va(rva::get().game_man)
+            .map_err(|_| fromsoftware_shared::InstanceError::NotFound)?
+            as *mut GameMan;
+
+        Ok(unsafe { &mut *target })
```

Comment:
If you use [`ptr::as_mut()`](https://doc.rust-lang.org/std/primitive.pointer.html#method.as_mut) here, you can avoid undefined behavior is the pointer is null.